lib/services/local/resource.go

```diff
@@ -43,6 +43,7 @@ func CreateResources(ctx context.Context, b backend.Backend, resources ...types.
 	if err != nil {
 		return trace.Wrap(err)
 	}
+<<<<<<< HEAD
```

Comment:
You have some unmerged code here.

Reply:
whoops didn't write my changes before `git cherry-pick --continue` 😅